DeepLinkKit/Router/DPLDeepLinkRouter.m

```diff
@@ -12,6 +12,8 @@ @interface DPLDeepLinkRouter ()
 @property (nonatomic, strong) NSMutableOrderedSet *routes;
 @property (nonatomic, strong) NSMutableDictionary *classesByRoute;
 @property (nonatomic, strong) NSMutableDictionary *blocksByRoute;
+@property (nonatomic, copy) DPLRouteHandlerBlock beforeEachBlock;
+@property (nonatomic, copy) DPLRouteHandlerBlock afterEachBlock;
```

Comment:
Are we sure we want to use copy here? I see we use `[routeHandlerBlock copy]` below so it's not enough?

Reply:
Yup `copy` here is required, I think we can drop the `copy` in `register` methods.